makefile_include.mk

```diff
@@ -92,7 +92,7 @@ endif
 ifeq ($(PLATFORM),FreeBSD)
   _ARCH := $(shell sysctl -b hw.machine_arch)
 else
-  _ARCH := $(shell arch)
+  _ARCH := $(shell sh -c "if command -v arch 1> /dev/null;then arch;else uname -m;fi")
```

Comment:
why not simply always `uname -m`?

Reply:
I checked if it is a link/alias which it isn't. If I would have taken the venturous enterprise of RTFM I would have seen "arch - print machine hardware name (same as uname -m)" right at the beginning. But who reads the fine manual?

Yepp, I shouldn't shoot from the hip so often, it's bad for the resume ;-)

And it gets worse. While trying to find out if there is a Posix way to get the machine architecture 
I stumbled upon this little gem: https://superuser.com/questions/835514/why-do-uname-p-and-uname-m-and-arch-output-different-architectures
Apple. \*sigh\*
(Doesn't matter now but might matter later)

So it's `uname -m` then.